packages/react-instantsearch/src/widgets/MenuSelect.js

```diff
@@ -27,6 +27,12 @@ import MenuSelectComponent from '../components/MenuSelect.js';
  *     >
  *       <MenuSelect
  *         attributeName="category"
+ *         transformItems={items =>
+ *           items.map(item => {
+ *             item.count = (item.count + 1000).toLocaleString();
```

Comment:
I'm not sure if this documentation is needed, it might be confusing why you're adding 1000

Reply:
Agree with @Haroenv, the examples in the documentation try to be the most minimal possible. For more elaborate example we have Storybook. No need to demonstrate the usage of transform Items on the documentation page.

Reply:
Thanks for the input. I'll remove it and push the commit.